server/src/main/java/app/friendship/FriendshipService.java

```diff
@@ -14,35 +13,37 @@ public class FriendshipService {
 
     public void sendFriendRequest(Integer senderId, Integer receiverId) {
         boolean alreadySent = friendshipRepository.existsBySenderIdAndReceiverId(
-            senderId,
-            receiverId
+                senderId,
+                receiverId
         );
 
         if (alreadySent) return;
 
         boolean canBeAcceptedInstantly = friendshipRepository.existsBySenderIdAndReceiverId(
-            receiverId,
-            senderId
+                receiverId,
+                senderId
         );
 
         if (canBeAcceptedInstantly) {

```

Comment:
<img width="1041" alt="Screenshot 2024-12-27 at 01 03 57" src="https://github.com/user-attachments/assets/e5d4156c-c2d0-46df-8c80-9ab9501c4231" />

your formatter has moved those lines. Please check if you have tab size eq to 4

if yes please use our project scheme (improt it from /competeme/server/config/intellij-code-style.xml )
<img width="1041" alt="Screenshot 2024-12-27 at 01 05 05" src="https://github.com/user-attachments/assets/3f3c662c-5bf4-4d00-80ff-95371601bb10" />